stores/user/index.js

```diff
@@ -29,7 +29,7 @@ const signedIn = (data) => ({
   userId: data.user.id
 })
 
-const signOut = () => ({
+const signOut = () => ({ 
```

Comment:
@hstove the [editorconfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) plugin will prevent these trailing spaces from showing up